<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="IO_box">
      <property name="title">
       <string>Input/Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="main_IO">
         <item>
          <layout class="QHBoxLayout" name="input_folder">
           <item>
            <widget class="QPushButton" name="image_input_folder_button">
             <property name="text">
              <string>Choose Input Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_input_folder_label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="output_folder">
           <item>
            <widget class="QPushButton" name="image_output_folder_button">
             <property name="text">
              <string>Choose Output Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="image_output_folder_label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="options_box">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="main_options">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="copy_images_chkbox">
             <property name="text">
              <string>Copy all Images</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="original_size_chkbox">
           <property name="text">
            <string>Keep original size</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
